src/main/java/com/techiteasy/demo/repositories/RemoteControllerRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.techiteasy.demo.repositories;
+
+import com.techiteasy.demo.models.RemoteController;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
die had ik vergeten op mijn eigen project

Reply:
Deze annotatie is niet nodig, omdat je al extend van JpaRepository. Daar (of in een van de diepere extensies) wordt deze annotatie al toegepast.